### PR TITLE
Added setup endpoint coverage

### PR DESCRIPTION
**Pull request by jppdpf**

### CODE REVIEW

File: sample/run_bit_sample.py [hunk 1]
```diff
@@ -1,6 +1,17 @@
...
 from os import getenv
-from yk_bit import BaseUrl, Key, capture, verify_images, verify, status, BiTStatus
+from yk_bit import BaseUrl, Key, capture, verify_images, verify, status, setup, BiTStatus
+
+"""
+First time running:
+ - Run the BiT App
+ - Run this SDK App
+ - If the setup was successful: 
+   - Restart the BiT App
```
Comment (dev-yoonik) [inline sample/run_bit_sample.py:10]: This restart and repeat logic are not needed.

File: yk_bit/bit.py [hunk 2]
```diff
@@ -102,3 +102,16 @@ def status() -> BiTStatus:
...
+    """
+        Perform BiT setup actions
+    :return: True if setup was successful, False otherwise
+    """
+    url = 'bit/setup'
+    try:
+        utils.request('GET', url)
+    except Exception:
+        raise
+    return True
```
Comment (vitormcpedro) [inline yk_bit/bit.py:117]: This function never returns False. If we are raising an exception when an error occurs, we do not need to return a value.

**Review by vitormcpedro — CHANGES_REQUESTED** [review]

File: yk_bit/bit.py (hunk 2 above)
Comment (mlourencoeb) [reply yk_bit/bit.py:118]: Setup logic with true/false seems to ugly. if the setup does not work, just let the exception being raised.

**Review by dev-yoonik — APPROVED** [review]

File: sample/run_bit_sample.py [hunk 3]
```diff
@@ -17,6 +26,17 @@ def base64_to_file(filename: str, data: str):
 
 
 if __name__ == "__main__":
+
+    # Setup
+    bit_setup = False
+    try:
+        bit_setup = setup()
```
Comment (vitormcpedro) [inline sample/run_bit_sample.py:33]: setup() method is not returning a value right now

File: sample/run_bit_sample.py [hunk 4]
```diff
@@ -17,6 +26,17 @@ def base64_to_file(filename: str, data: str):
...
+    except Exception as ex:
+        print(ex)
+    finally:
+        print(f"BiT Setup Successful: {bit_setup} \n")
+        exit()
```
Comment (vitormcpedro) [inline sample/run_bit_sample.py:38]: why exiting here?

**Review by vitormcpedro — CHANGES_REQUESTED** [review]

**Review by vitormcpedro — APPROVED** [review]

**Pull request merged**